source/core.py

```diff
@@ -419,18 +436,9 @@ def handlePowerStatusChange(self):
 	# initialize wxpython localization support
 	locale = wx.Locale()
 	lang=languageHandler.getLanguage()

```

Comment:
```suggestion
	lang: str = languageHandler.getLanguage()

```

Or just  set the return type for `getLanguage`

Reply:
done the alternative of setting the return type for `getLanguage`